### DISCUSSION TITLE
Vite5 uses tailwind4 and the configuration is invalid

### DISCUSSION THREAD

**Discussion by yibird** [tailwindlabs/tailwindcss#13224] [Help]
I executed the following command based on the tailwind blog
```shell
npm install tailwindcss@next @tailwindcss/vite@next
```
Added tailwind plugin to vite.config.ts and introduced tailwind style in the entry file. I created a tailwind.config.js in the root directory of the project, and the configuration is as follows:
```js
/** @type {import('tailwindcss').Config} */
export default {
  theme: {
    spacing: {
      '32': '100px',
    },
  },
};
```
I used styles like H-32 W-32 and actually used height: 8rem; width: 8rem; This means that the configuration is invalid and I don't know what to do

![image](https://github.com/tailwindlabs/tailwindcss/assets/42264628/0775d288-6bbb-4a23-b378-b3ff22023eb4)

**Comment by wongjn** [comment]
As of time of writing of the latest release `4.0.0-alpha.8`, v4 does **not** use `tailwind.config.js` but there is a plan to add some sort of JavaScript configuration interface.

At the moment, configuration is solely done in the source CSS file, so the equivalent of your configuration would be:

```css
@import 'tailwindcss';

@theme {
  --spacing-32: 100px
}
```
